PowerEditor/src/NppDarkMode.cpp

```diff
@@ -1582,7 +1584,7 @@ namespace NppDarkMode
 
 		case WM_NCDESTROY:
 		{
-			RemoveWindowSubclass(hWnd, TabSubclass, g_tabSubclassID);
+			::RemoveWindowSubclass(hWnd, TabSubclass, uIdSubclass);
```

Comment:
Are you sure about it?
The current code in master we have symetric calls `SetWindowSubclass(hwnd, TabSubclass, g_tabSubclassID, 0);` & `RemoveWindowSubclass(hWnd, TabSubclass, g_tabSubclassID);`.
In this PR  I wonder if `g_tabSubclassID` is still useful.

Reply:
Yes.
`SetWindowSubclass` will send `g_tabSubclassID` via `TabSubclass` (https://learn.microsoft.com/en-us/windows/win32/api/commctrl/nf-commctrl-setwindowsubclass)

IIRC there was error with some subclass which had wrong subclass id variable, probably due to copy&paste. Fortunately every subclass ids Notepad++ use have same value.

So this is mainly to avoid future issue when copy&paste or if several subclassing is required.